discovery-scripts/windows/smartctl-disks-discovery.ps1

```diff
@@ -1,53 +1,58 @@
-# VERSION = 1.5
+# VERSION = 1.6
```

Comment:
VERSION in perl script and userparams file should be changed as well then.

Reply:
correct